draft-ietf-rats-architecture.md

```diff
@@ -734,10 +734,29 @@ or interval when changes in operational status, health, and so forth occur.
 
 ## Endorsements
 
-An Endorsement is a secure statement that some entity (e.g., a manufacturer) vouches for the integrity of the
-device's signing capability.  For example, if the signing capability is in hardware, then
-an Endorsement might be a manufacturer certificate that signs a public key whose corresponding
-private key is only known inside the device's hardware.  Thus, when Evidence and such an Endorsement
+An Endorsement input to a Verifier is always required. An Endorsement is what allows a Verifier to trust an Attester and in turn to trust the Evidence produced by an Attester. Without Endorsements a Verifier cannot function.
+Said another way, an Attester for which there is no Endorsement is not useful since there is no good basis by which a Verifier can trust it.
```

Comment:
I still have the same objection as before to this text.   (And I don't think new text is required either.)

Reply:
Do you think that Endorsements are not always required or do you agree with me that they are always required but the text here is redundant?

Reply:
Not always required.  The relevant part of my earlier comment above is "If a Verifier owner has some means to establish trust in each Attester's hardware individually then it need not use an endorsement."

Reply:
The trusting of an Attester by a Verifier is to me one of the most fundamental things about attestation and I think it needs to be address prominently and clearly in the architecture document.

I don't know of anyway practical way to establish this trust other than with crypto which of course means some key material in the Attester and related key material in the Verifier.

If an Endorsement is not the one true way the Verifier gets the key it needs, then the architecture should indicate an input to the Verifier by which it does get the key.

I do take it that you agree that these are all valid ways for the Verifier to get the key
 - In an X.509 certificate
 - A CMS-encrypted flat file of keys
 - A flash drive from the Attester maker walked into the server room containing the Verifier
 - TLS-secured access to a database of millions of keys at the Attester manufacturer
 - Many more...

Another option here is to abandon the term Endorsement. It does seem to have some problematic preconception (I keep seem to be guessing wrong, probably because I don't go to the right TCG meetings). Instead of Endorsement these are the verifier inputs:
  - Keys
  - Known-good / Reference values
  - Implicit claims
  - Policy configuration, scripts and such.

Reply:
> The trusting of an Attester by a Verifier is to me one of the most fundamental things about attestation and I think it 
> needs to be address prominently and clearly in the architecture document.

Of course.

> I don't know of anyway practical way to establish this trust other than with crypto which of course means some key material in the Attester and related key material in the Verifier.

Yes.

> If an Endorsement is not the one true way the Verifier gets the key it needs, then the architecture should indicate an input to the Verifier by which it does get the key.

If you have an Endorsement, the Verifier needs a way to get the manufacturer's key and establish trust in it.
Without an Endorsement, that same mechanism applies to the Attester's key directly.
Endorsements just provide scalability since you can put one key in your trust anchor store rather than (say) 1 per Attester.

An Endorsement is simply one external key (typically a manufacturer but need not be) vouching for the Attesting Environment's key.

> I do take it that you agree that these are all valid ways for the Verifier to get the key
[...]

Yes

> Another option here is to abandon the term Endorsement.

I would disagree with that option.  The notion of an external key vouching for the Attesting Environment's key is a key concept.   As you state, it's not the only input to the Verifier but that doesn't mean the term isn't an important concept to explain.